### PR TITLE
Adding 1 site

### PR DESCRIPTION
**Pull request by C3n7ral051nt4g3ncy**
## Added 1 site:

- `Admire_me `

### CODE REVIEW

**Comment by WebBreacher** [comment]
@C3n7ral051nt4g3ncy since the `Create sync_upstream_wmn.yml` is a file you have created for your system and does not impact the project, I'm going to ask that you not add that file name to the project's `.gitignore` file. You may need to move this file out of the WMN directory or do something else so as to not submit the file.

Please remove the change to the project's `.gitignore`

File: wmn-data.json [hunk 1]
```diff
@@ -130,7 +130,18 @@
         "m_code" : 200,
         "known" : ["mike", "greg"],
         "cat" : "coding",
-        "valid" : true
+        "valid" : true       },
```
Comment (WebBreacher) [inline wmn-data.json:133]: Seeing a Ctrl-M here again. Please remove.
image

Comment (C3n7ral051nt4g3ncy) [reply wmn-data.json:133]: @WebBreacher:
- Ctrl-M `removed`
- gitignore additions `removed`

Comment (WebBreacher) [reply wmn-data.json:133]: The Ctrl-M persists. I'll accept the PR and then manually change it.

**Pull request merged**